csharp/src/Ice/OutgoingResponseFrame.cs

```diff
@@ -161,8 +163,14 @@ public static OutgoingResponseFrame WithVoidReturnValue(Current current)
                         }
                         else
                         {
-                            Data[0] = Data[0].Slice(0, 1);
+                            // We are forwarding a system exception encoded using 1.1 encoding and ice2 protocol over
```

Comment:
We are forwarding a system exception encoded using 1.1 and received over ice2 to ice1.